models/MythoMax-L2-13B.json

```diff
@@ -2,7 +2,7 @@
     "_descriptorVersion": "0.0.1",
     "datePublished": "2023-08-10T20:51",
     "name": "MythoMax L2 13B",
-    "description": "An improved, potentially even perfected variant of MythoMix, MythoLogic-L2 and Huginn merge using a highly experimental tensor type merge technique. The main difference with MythoMix is that it was allowed more of Huginn to intermingle with the single tensors located at the front and end of a model, resulting in increased coherency across the entire model. This model is proficient at both roleplaying and storywriting due to its unique nature. Using MythoLogic-L2's robust understanding as its input and Huginn's extensive writing capability as its output seems to have resulted in a model that exceeds at both",
+    "description": "MythoMax-L2-13B is an exceptional creative writing model that harnesses the power of Llama2, MythoLogic-L2, and Huginn. It can craft excellent and captivating texts for various domains and audiences, using MythoLogic-L2’s vast knowledge base and Huginn’s expressive language skills. It is also proficient at roleplaying, able to adapt to different contexts and characters with flexibility and realism. This model is a superb solution for creative writing or roleplaying.",
```

Comment:
```suggestion
    "description": "MythoMax-L2 is a creative writing model that builds upon Llama2, MythoLogic-L2, and Huginn. It can craft captivating texts for various domains and audiences, using MythoLogic-L2’s vast knowledge base and Huginn’s expressive language skills. Specialized at creative writing, it is also proficient at roleplaying, and is able to adapt to different contexts and characters with flexibility and realism. This model is an experimental merge of several other models (MythoMix, Mythologic-L2, and Huggin).",
```